classes/class-wc-connect-nux.php

```diff
@@ -617,7 +617,12 @@ public function show_nux_banner( $content ) {
 			?>
 			<div class="notice wcs-nux__notice <?php echo isset( $content['dismissible_id'] ) ? 'is-dismissible' : ''; ?>">
 				<div class="wcs-nux__notice-logo">
-					<img src="<?php echo esc_url( $content['image_url'] );  ?>">
+					<?php if ( $content['should_show_jp'] ) : ?>
+						<img class="wcs-nux__notice-logo-jetpack" src="<?php
+						echo esc_url( plugins_url( 'images/jetpack-logo.png', dirname( __FILE__ ) ) );
+						?>">
```

Comment:
Can we collapse this to one line? This will result in newlines inside the `src` attribute.